_includes/troubleshoot/issues/nopods-in-daemonset.md

```diff
@@ -0,0 +1,62 @@
+## 0/0 Pods in DaemonSet
+
+### Issue:
+
+StorageOS pods not being scheduled and no error is shown in the DS.
+```bash
+{{ page.cmd }} -n storageos get daemonset
+NAME                  DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE NODE SELECTOR   AGE
+storageos-daemonset   0         0         0         0            0         <none>          5s
+```
+
+### Reason:
+This indicates that no node in the cluster was selected by the Operator to
+deploy StorageOS.
+
+### Assert
+Check the NodeSelector section of the CustomResource.
+
+```bash
+{{ page.cmd }} -n default describe stos storageos
+
+...
+
+Node Selector Terms:
+Match Expressions:
+  Key:       node-role.kubernetes.io/worker
+  Operator:  In
+  Values:
+    wrongValue <-- No node in the cluster has this label
+...
+```
+
+> Where the CR name is "storageos"
```

Comment:
s/CR/CustomResource/

"Where the CustomResource name in the {{ page.cmd }} instantiation is 'storageos'"